cpu/sam0_common/periph/uart.c

```diff
@@ -80,9 +80,11 @@ int uart_init(uart_t uart, uint32_t baudrate, uart_rx_cb_t rx_cb, void *arg)
                       SERCOM_USART_CTRLA_SAMPR(0x1) |
                       SERCOM_USART_CTRLA_TXPO(uart_config[uart].tx_pad) |
                       SERCOM_USART_CTRLA_RXPO(uart_config[uart].rx_pad) |
-                      SERCOM_USART_CTRLA_MODE(0x1) |
-                      (uart_config[uart].runstdby ?
-                              SERCOM_USART_CTRLA_RUNSTDBY : 0));
+                      SERCOM_USART_CTRLA_MODE(0x1));
+    /* Set run in standby mode if enabled */
+    if ((uart_config[uart].flags & UART_RUN_STANDBY) == UART_RUN_STANDBY) {
```

Comment:
can be shortened to ` if (uart_config[uart].flags & UART_RUN_STANDBY) {`

Reply:
Ok I will make the change, I know some prefer the extra comparison, but that does not appear to be the case with RIOT code.